.infra/workers.ts

```diff
@@ -215,4 +215,12 @@ export const workers: Worker[] = [
     topic: 'analytics-api.v1.experiment-allocated',
     subscription: 'api.experiment-allocated',
   },
+  {
+    topic: 'api.v1.source-created',
+    subscription: 'api.source-created-squad',
+  },
+  {
+    topic: 'api.v1.source-created',
+    subscription: 'api.source-created-squad-mailing',
```

Comment:
Perhaps also squad-owner-mailing to be more explicit as to what this subscription handles?